reviews/templates/reviews/review_detail_sidebar.html

```diff
@@ -77,6 +77,15 @@ <h3>
                 </em>
             </li>
         {% endif %}
+        {% if review.stage == review.SUPERVISOR %}
+            <li>
+                <strong>
+                    {% blocktrans trimmed %}
+                        Dit is een review van een eindverantwoordelijke
```

Comment:
Maybe better to use "beslissing"? I don't think we use "review" in user-facing text elsewhere.

Reply:
Good point, I'll change this when I rebase this branch on the latest develop. (There seems to be a merge conflict at the moment)